editor/question-ctrl.js

```diff
@@ -150,42 +152,45 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
-  		$scope.questionData.questionTitle = metaFormScope.contentMeta.name;
-  		$scope.questionData.qcMedium = metaFormScope.contentMeta.medium;
-  		$scope.questionData.qcLevel = metaFormScope.contentMeta.level;
-  		$scope.questionData.questionDesc = metaFormScope.contentMeta.description;
-  		$scope.questionData.questionMaxScore = metaFormScope.contentMeta.max_score;
-  		$scope.questionData.qcGrade = metaFormScope.contentMeta.gradeLevel;
-  		$scope.questionData.concepts = metaFormScope.contentMeta.concepts;
-  		$scope.questionData.topic = metaFormScope.contentMeta.topic;
+      metaFormScope.isSubmit = false;
+  		for (var key in metaFormScope.contentMeta) {
+        if (metaFormScope.contentMeta.hasOwnProperty(key)) {
+          if (key == 'name') {
+            $scope.questionData['title'] = metaFormScope.contentMeta['name'];
+          } else {
+            $scope.questionData[key] = metaFormScope.contentMeta[key];
+          }
+        }
+       }
+
   		$scope.questionMetadataScreen = false;
   	}
   }
   $scope.formIsValid = function () {
   	$scope.questionMetadataScreen = true;
     //comment because in edit question the question and question title are not
     if ($scope.category == 'FTB') {
-    	$scope.questionData.questionTitle = _.isUndefined($scope.questionData.questionTitle) ? $scope.questionCreationFormData.question.text.replace(/\[\[.*?\]\]/g, '____') : $scope.questionData.questionTitle;
-    } else {
-    	$scope.questionData.questionTitle = _.isUndefined($scope.questionData.questionTitle) ? $scope.questionCreationFormData.question.text : $scope.questionData.questionTitle;
-    }
-    $scope.questionData.questionTitle = $scope.extractHTML($scope.questionData.questionTitle);
-    $scope.questionMetaData.title = $scope.questionData.questionTitle;
-    $scope.questionMetaData.medium = $scope.questionData.qcMedium;
-    $scope.questionMetaData.qlevel = $scope.questionData.qcLevel;
-    $scope.questionMetaData.description = $scope.questionData.questionDesc;
-    $scope.questionMetaData.max_score = $scope.questionData.questionMaxScore;
-    $scope.questionMetaData.gradeLevel = $scope.questionData.qcGrade;
-    $scope.questionMetaData.concepts = $scope.questionData.concepts;
-    $scope.questionMetaData.topic = $scope.questionData.topic;
-    $scope.questionMetaData.subject = $scope.questionData.subject;
-    $scope.questionMetaData.board = $scope.questionData.board;
-    if ($scope.questionMetaData.concepts) {
-    	$scope.questionMetaData.conceptData = "(" + $scope.questionData.concepts.length + ") concepts selected";
-    }
-    if ($scope.questionMetaData.topic) {
-      $scope.questionMetaData.topicData = "(" + $scope.questionData.topic.length + ") topics selected";
-    }
+        $scope.questionData.title = _.isUndefined($scope.questionData.title) ? $scope.questionCreationFormData.question.text.replace(/\[\[.*?\]\]/g, '____') : $scope.questionData.title;
```

Comment:
Can see too much logic to handle "title" here. Lets discuss & handle it properly.